### ISSUE TITLE
Super bold window frame when using 175% global scaling 

### ISSUE THREAD

**Issue by adam900710** [PapirusDevelopmentTeam/materia-kde#127]
When "Window decoration" theme is changed to any Materia theme, and with global scaling other than 100%, it will result super bold window frame, definitely way larger than the scaled result.

Default Oxygen has no such problem.

![2021-10-07-143315_2560x1440_scrot](https://user-images.githubusercontent.com/460269/136331866-9c22f089-7662-486e-980f-b89760417bf0.png)

**Comment by varlesh** [comment]
Try set without borders on Window border size

**Comment by varlesh** [comment]
What you mean `No border is too aggressive then.`?

**Comment by varlesh** [comment]
Your issue specific... but you can play with borders size...
Open Materia-Darkrc and replace `Layout` section:
```
[Layout]
BorderBottom=0
BorderLeft=0
BorderRight=0
ButtonHeight=24
ButtonMarginTop=0
ButtonSpacing=6
ButtonWidth=24
ExplicitButtonSpacer=0
PaddingBottom=58
PaddingLeft=26
PaddingRight=26
PaddingTop=10
TitleBorderLeft=4
TitleBorderRight=4
TitleEdgeBottom=4
TitleEdgeBottomMaximized=4
TitleEdgeLeft=8
TitleEdgeLeftMaximized=4
TitleEdgeRight=8
TitleEdgeRightMaximized=4
TitleEdgeTop=4
TitleEdgeTopMaximized=4
TitleHeight=12
```
restart kwin:
`killall kwin_x11`
ALT+F2 > `kwin_x11`

**Comment by varlesh** [comment]
please read my previous comment

**Comment by varlesh** [comment]
I cannot figure out how it works. In any case, there is one pixel left around the border of the windows.

**Comment by varlesh** [comment]
And it will be difficult to describe as a KWin bug.

**Comment by varlesh** [comment]
Need redraw and reduce shadows... but i'm not want doing this, i spent a lot of time aligning with the GTK theme.
I think, better temporary solution for you - use Breeze theme

**Comment by varlesh** [comment]
Funnily enough, in the Wayland session, Aurorae also has scaling issues. https://github.com/PapirusDevelopmentTeam/materia-kde/issues/92 https://bugs.kde.org/show_bug.cgi?id=422342
Unsurprisingly, this is a very old engine that was first released 12 years ago.

**Comment by adam900710** [comment]
Fair enough, would use something like Arc for systems with fractional scale as a workaround for now.

**Comment by nekohasekai** [comment]
It also exists on my 2560/1600 device with 150% zoom.
By the way, there is no problem under wayland.

**Comment by demfabris** [comment]
Also experiencing this. This is not the only theme to have chunky borders with fractional scaling
